activemq-karaf/src/main/resources/features.xml

```diff
@@ -39,7 +39,7 @@
 
     <feature name="activemq-camel" version="${upstream.version}" resolver="(obr)" start-level="50">
         <feature version="${upstream.version}">activemq-client</feature>
-        <bundle>mvn:org.apache.activemq/activemq-camel/${upstream.version}</bundle>
+        <bundle>mvn:org.apache.activemq/activemq-camel/${activemq-camel.tesb.version}</bundle>
```

Comment:
@xldai , version update is done here already

Reply:
Yep, my fault.
then LGTM.